packages/expo-updates/ios/EXUpdates/EXUpdatesService.m

```diff
@@ -70,6 +70,14 @@ - (BOOL)isUsingEmbeddedAssets
   return EXUpdatesAppController.sharedInstance.isUsingEmbeddedAssets;
 }
 
+- (BOOL)isEmbeddedLaunch
+{
+  return ([self embeddedUpdate] != nil &&
+          [self launchedUpdate] != nil &&
+          [[self embeddedUpdate].updateId.UUIDString isEqualToString:[self launchedUpdate].updateId.UUIDString]
```

Comment:
I believe you can just do equality on the UUID instances themselves rather than doing the extra conversion to string:
```suggestion
          [[self embeddedUpdate].updateId isEqual:[self launchedUpdate].updateId]
```

Reply:
Agreed -- pushed a simpler implementation of this method